### ISSUE TITLE
Change the default tool to zoom

### ISSUE THREAD

**Issue by MarcSkovMadsen** [holoviz/hvplot#837]
I see over and over again that users gets confused that the default tool is pan and not zoom.

Normally the plots are zoomed out so that you can see anything. The first action you want to do (if any) is to zoom into that interesting area you have found.

Its simply not a good user experience to have to first select a different tool and then do what you want. It confuses beginning users and slows you down as an experienced user. It makes it harder to mix with other plotting frameworks as they typically enable zoom by default.

**Comment by jbednar** [comment]
I agree; zoom should be enabled by default! But note that Pan and Zoom are not actually alternatives; both can be active at once since they use different events:

<img width="317" alt="image" src="https://user-images.githubusercontent.com/1695496/185462543-c4844851-9cf0-4bc8-a09b-66d5c28f3df4.png">

So my vote is for both to be active by default, not just Pan.

**Comment by MarcSkovMadsen** [comment]
Thanks. I was actually thinking of box zoom. I don't think pan and box zoom can be active at the same time?

**Comment by jbednar** [comment]
Right; box and pan use the same events. So I'd vote against that, because making Box enabled makes Pan not enabled, while making Scroll Zoom doesn't remove the Pan capabilities, and thus seems like a more powerful default. But maybe I'm biased because I got out of the habit of using box zoom back when Bokeh couldn't enforce an aspect ratio and it messed up everything,  so I stopped even trying it. Not sure how typical I am in that respect.

**Comment by MarcSkovMadsen** [comment]
I've also tried the `pan` + `wheel_zoom` combination. The pro of that combination is 1) You almost never have to change tool 2) It would not be a "breaking" change. The con is that other plotting frameworks like Plot uses box-zoom by default. So users often expect that.

**Comment by maximlt** [comment]
> The con is that other plotting frameworks like Plot uses box-zoom by default

You mean *Plotly*? And what other frameworks have this default behavior?

I'm more used to Pan + Wheel Zoom as a default, as you get on Google Maps for instance, it feels very natural. The only thing I'd like to check is how Bokeh Wheel Zoom tools behave on mobile.

**Comment by jlstevens** [comment]
I do think that pan+wheel_zoom is the most sensible default. 

It looks like by default HoloViews only activates the `pan` tool: I would be happy to see the `wheel_zoom` also activated by default and I don't think this should really break backwards compatibility for anyone. Any thoughts @jbednar @philippjfr ?
